docs/guides/modules/authorisation.rst

```diff
@@ -9,11 +9,19 @@ Installation
 You will need to setup the Users and Data Table services in your app:
 
   * Ensure that you have added the 'Users' service to your app
+
+Classic Mode (requires changes to Users table):
   * In the 'Data Tables' service, add:
   	* a table named 'permissions' with a text column named 'name'
 	* a table named 'roles' with a text column named 'name' and a 'link to table'column named 'permissions' that links to multiple rows of the permissions table
 	* a new 'link to table' column in the Users table named 'roles' that links to multiple rows of the 'roles' table
 
+Custom Mode (no changes to Users table):
+  * In the 'Data Tables' service, add:
+  	* a table named 'permissions' with a text column named 'name'
+	* a table named 'roles' with a text column named 'name' and a 'link to table'column named 'permissions' that links to multiple rows of the permissions table
+	* a table with a 'link to table' column named 'user' that links to a single row of the 'users' table and a 'link to table' column named 'roles' that links to multiple rows of the 'roles' table
+
```

Comment:
```suggestion
  * In the 'Data Tables' service, add:
  	1. a table named 'permissions' with a text column named 'name'
	2. a table named 'roles' with a text column named 'name' and a 'link to table'column named 'permissions' that links to multiple rows of the permissions table
	3. a new 'link to table' column in the Users table named 'roles' that links to multiple rows of the 'roles' table (This behaviour is configurable - see below)

Configuration Options:
++++++++++++++++++++++

Certain default behaviour is configurable

By default the user's table expects a 'roles' column, that links to multiple rows of the 'roles' table.
Alternatively you can create a table that maps a user to linked rows of the role's table, avoiding adding extra columns to the user's table.

* Create a table called 'usermap' with a column 'user' and a column 'roles'.
* the 'user' column is a link to a row in the user's table. The 'roles' column is a link to multiple rows from the 'roles' table.

.. code-block:: python

    import anvil.server
    from anvil_extras.authorisation
    
    authorisation.set_config(get_roles="usermap")

Now the authorisation module will use the 'usermap' table to get roles for a user.


```

I'd maybe add the Configuration options just above API
And then add `set_config` to the API section